artifacts/definitions.py

```diff
@@ -34,6 +35,8 @@
     'Mail': 'Mail client applications artifacts.',
     'Memory': 'Artifacts retrieved from memory.',
     'Network': 'Describe networking state.',
+    'Persistence': 'Artifact location potentially used for malware persistence.',
```

Comment:
Maybe: `Artifact location potentially used for malware persistence.` => `Artifact known to be used for persistence of malware.`

Reply:
agree